src/main/resources/templates/notifications.ftl

```diff
@@ -0,0 +1,53 @@
+[#import "macros.ftl" as macros]
+[@macros.renderHeader i18n.translate("section.notifications") /]
+
+[@macros.renderMenu i18n user /]
+<div class="container">
+    <h2>
+    ${i18n.translate("block.my-notifications.title")}
+        <div class="pull-right"></div>
+    </h2>
+[#if error?? && error?has_content]
+    <div class="alert alert-danger">
+    ${i18n.translate(error)}
+    </div>
+[/#if]
+    <table class="table table-bordered">
+        <tbody>
+[#if notifications??]
+    [#list notifications as notificationWithUser]
+        <tr>
+            [#if notificationWithUser.isRead()]
+            <td class="notification read">
+            [#else]
+            <td class="notification unread">
+            [/#if]
+                [#if !notificationWithUser.isRead()]
+                <form action="${path}/markRead" method="post" class="pull-right">
+                    <input type="hidden" name="notificationId" value="${notificationWithUser.getNotification().getId()}">
+                    <button type="submit" class="btn btn-danger btn-sm" style="margin: 5px;">
+                        <i class="glyphicon glyphicon-remove-sign"></i> ${i18n.translate("block.my-notifications.mark-read")}
+                    </button>
+                </form>
+                [#else]
+                <span class="pull-right">
+                    <i class="pull-right">${i18n.translate("block.my-notifications.is-read")}</i>
+                </span>
+                [/#if]
+                <div>${notificationWithUser.getNotification().getSender().getName()}, ${notificationWithUser.getNotification().getEvent()} #${notificationWithUser.getNotification().getId()}</div>
```

Comment:
Alright, this may sound nitpicky but wouldn't it be an idea to display the issue id instead of the notification id? Displaying the latter seems rather arbitrary as it doesn't relate to the issue itself, whereas the issue id would be more relevant to the user as the notification would then clearly indicate the issue it's related to.

![image](https://user-images.githubusercontent.com/23633658/27263659-4be9417e-546e-11e7-87a1-8624dd9954f0.png)
![image](https://user-images.githubusercontent.com/23633658/27263669-7b5a8242-546e-11e7-9646-ef24aa063aed.png)

Like here it would be nice if the id displayed in the notification would correspond to the id of the actual issue.